circle.yml

```diff
@@ -0,0 +1,17 @@
+dependencies:
+  cache_directories:
+    - /usr/local/Homebrew
```

Comment:
Will this cache the homebrew installs we make when running the script? Or only after the setup script step?